src/plugins/pull-request.ts

```diff
@@ -61,6 +65,18 @@ export const createPullRequest = async (
       return true;
     }
 
+    const { stdout: currentBranch } = await asyncExec(
+      "git log --pretty=oneline -1"
+    );
+    const { stdout: mainBranch } = await asyncExec(
+      "git log --pretty=oneline -1 main"
+    );
+
+    if (currentBranch.split(" ")[0] === mainBranch.split(" ")[0]) {
```

Comment:
Just a suggestion based on some research today.

```suggestion
    const { stdout: currentBranch } = await asyncExec(
      "git rev-parse HEAD"
    );
    const { stdout: mainBranch } = await asyncExec(
      "git rev-parse main"
    );

    if (currentBranch === mainBranch) {
```

---

🔧 I realise this is only (currently) used for `freeCodeCamp/freeCodeCamp`, but what if this were used on a repo where the default branch was not named `main`...

_Feel free to ignore all my comments, and just merge this, because it should just work.

Reply:
Suppose I could use the `base` parameter that indicates the branch the PR will target 🤦🏻‍♀️